SECURITY.md

```diff
@@ -3,12 +3,12 @@
 ## Reporting a Vulnerability
 
 
-To report a Security Vulnerability or issue, we ask that you please email any applicable information to: bots[at]miraheze.org.
+To report a Security Vulnerability or issue, we ask that you please email any applicable information to: staff[at]mirahezebots.org.
 
 Please note that by doing this you agree to disclose your email address, and/or any information provided in the email header.
 
 At a minimum, we ask that you provide, context to the finding of the vulnerability/issue, steps to recreate the vulnerability/issue, and if you have a potiental solution.
 
 Please do note every effort will be done to respect your privacy, however, please do note the following:
 
-By using this project or contacting us, you agree that your details can be used in accordance with our Privacy Policy (https://bots.miraheze.wiki/privacy.html).
```

Comment:
Use /privacy

Reply:
> Use /privacy

@RhinosF1 Well `privacy.html` is the actual name of the page, I would think, though we may have a web redirect/alias set up for `privacy`. Isn't this mainly a cosmetic request, though?